.github/workflows/tests.yaml

```diff
@@ -83,11 +83,19 @@ jobs:
           micromamba list
 
       - name: Test
+        continue-on-error: ${{ matrix.dev }}
```

Comment:
Not a big fan of this. If a test fails without CoW but passes with it, we will most likely not notice because the check will be green.

Reply:
I am not sure if it is worth splitting this into a separate CI run but can't think of a better solution.